sdk/openfeature-nodejs-provider/project.json

```diff
@@ -10,18 +10,25 @@
                 "outputPath": "dist/sdk/openfeature-nodejs-provider",
                 "tsConfig": "sdk/openfeature-nodejs-provider/tsconfig.lib.json",
                 "packageJson": "sdk/openfeature-nodejs-provider/package.json",
-                "updateBuildableProjectDepsInPackageJson": true,
-                "buildableProjectDepsInPackageJsonType": "dependencies",
                 "main": "sdk/openfeature-nodejs-provider/src/index.ts",
                 "assets": ["sdk/openfeature-nodejs-provider/*.md"],
-                "external": ["nodejs", "shared-types"]
+                "external": [
+                    "nodejs",
+                    "shared-types",
+                    "js-cloud-server-sdk",
+                    "server-request",
+                    "config-manager"
```

Comment:
are these both externals?

Reply:
no but they need to be listed here because otherwise it bundles them in here as well, even though they're bundled in the nodejs package